09 Environment/RiskAnalysis.md

```diff
@@ -0,0 +1,71 @@
+# Risk Analysis
+
+We use the following metric to illustrate severity:
+
+    1-2  : very low  
+    3-4  : low  
+    5-6  : medium  
+    7-8  : high  
+    9-10 : very high  
+
+
+|  Risk | Probability | Consequence | Priority | Revised Probability | Revised Consequence |
+| ------------------------------------ | ---------------------------------------------------- | --------- | --------- | ------------------ |--------- |
+| Unable to deliver complete app due to priorities | 3 | Unable to live up to client expectations | 5 | 2 | Unable to deliver report |
```

Comment:
You may need to look at the first priority, cause not going to the exam may be a very big problem.